notebooks/icrp107_dataset/icrp107_dataset.ipynb

```diff
@@ -1205,20 +1367,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 14,
    "metadata": {},
    "outputs": [],
    "source": [
-    "rows = np.array([], dtype=np.int)\n",
-    "cols = np.array([], dtype=np.int)\n",
-    "data = np.array([], dtype=np.float)\n",
+    "rows = np.array([], dtype=np.int64)\n",
+    "cols = np.array([], dtype=np.int64)\n",
+    "data = np.array([], dtype=np.float64)\n",
     "ln2 = np.log(2)\n",
     "\n",
     "for parent in nuclide_list:\n",
     "    j = nuclide_list.index(parent)\n",
     "    rows = np.append(rows, [j])\n",
     "    cols = np.append(cols, [j])\n",
-    "    lambd = ln2/icrp.at[parent, \"Half_life_s\"]\n",
+    "    half_life = icrp.at[parent, \"Half_life_s\"]\n",
+    "    if half_life == np.inf:\n",
+    "        lambd = 0\n",
```

Comment:
Maybe use `0.0` instead of `0` here. In more low-level programming languages like C & C++, it's often important to be explicit about the differences between int and float data types. Pure Python programmers might not care about this (Python is pretty robust with its dynamic typing), but being explicit is a good habit to get into!